### ISSUE TITLE
Linux Issues

### ISSUE THREAD

**Issue by peardox** [Embarcadero/PythonEnviroments#18]
I've been having problems with Embedded on Linux (told you about this before) - specifically Pip failing owing to no SSL support (and the machine has SSL of course)

Anyway - today I grabbed https://github.com/lmbelo/python3-embeddable and built a 3.9.12 on the target and replaced the existing non-functional 3.9 with the one I'd just compiled

Everythinng worked perfectly.... This begs the question - why?

Is my build environment different ot something (Ubuntu 22.04)?

Steam deck should be available in a month (or two) and Jim McKeeth _may_ be sorting something out for a D11 so I was just doing some exploratory work...

I'm also wondering if this may help the OSX builds....

**Comment by lmbelo** [comment]
Yes, I remember. I told you so, you can easily install openssl with apt-get. 
We are compiling the [macOS version with ssl](https://github.com/lmbelo/python3-embeddable/blob/7a1f7a236b94ad724916d8e6f6dad6a5f3d81463/python3-macos/MacOS/build-installer.py#L251), but Linux not.
I know it would be good to get it compiled with the Linux distribution. 
I will give it an attention as possible.

**Comment by peardox** [comment]
If you want a proper Linux server I've still got a load of AWS credits so I can let you have an x86 to build on.

Note - this will be a LOT easier than the problems you had with the AWS OSX  box - you just ssh into the box and do what you like. If you want me to set one up just tell me what OS you want on it, how much HD and suchlike. 

Ooh - they've got a data center in Sao Paulo - looks like I can sort you out a 4 CPU 16Gb RAM build box no problem (only costs about 15c an hour + disk space @ a buck per 10G/mth)

**Comment by peardox** [comment]
Just tried Sao Paulo out - created a 4CPU/16G with 20GB Disk with Ubuntu 22.04 - pricing is more expensive for some reason - 33c per hour.

Anyway - no problem creating one (I stopped it again after a min anyway - just testing) so if you want one for a day it'll only cost 10 bucks of my credit (I've got plenty left so no problem)

**Comment by lmbelo** [comment]
I can't jump into this task right now. I can get it working out on a local machine. We only need to uninstall openssl to get that error.

**Comment by peardox** [comment]
Steam Deck now available to purchase message (there's a up to 3 month queue) just came through so it's now ordered and "on the way"

Having worked out the way to get P4D working on Linux now I should be able to get an install on the Deck when it arrives specially built for the Deck and with PyTorch GPU support
